Resources/Textures/_DV/Clothing/Neck/Misc/cosmeticanomalies.rsi/meta.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "",
+    "size": {
+      "x": 32,
+      "y": 32
+    },
+    "states": [
+      {
+        "name": "icon"
+      },
+      {
+        "name": "equipped-NECK",
+        "directions": 4,
+        "delays": [
+          [
+            0.8,
+            0.8,
+            0.8,
+            0.8
+          ],
+          [
+            0.8,
+            0.8,
+            0.8,
+            0.8
+          ],
+          [
+            0.8,
+            0.8,
+            0.8,
+            0.8
+          ],
+          [
+            0.1,
+            0.1,
+            0.1,
+            0.1
+          ]
+        ]
+      }   
+    ]
+  }
```

Comment:
4 space indent dont mix it with 2
the rsi validator is failing on the 0.1s at the end, change them to 0.8 like the rest

also missing trailing newline